.gitignore

```diff
@@ -51,8 +51,6 @@ typings/
 # Output of 'npm pack'
 *.tgz
 
-# Output of `yarn build`
-lib
```

Comment:
Is this a typo?

Reply:
No, I just got my PR aimed at the wrong repo. :grimacing: Apologies, I was meaning to pull your upstream commits down to our fork of `discv5`.  Will close.